libs/README.md

```diff
@@ -17,7 +17,7 @@ downward line means "is required by"):
 We recommend building with conan 2, and only conan 2 builds are used on the CI.
 However, the libraries may be built using either conan 1 or conan 2.
 
-The build requires `cmake` version 3.26 or above.
+The build requires `cmake` version 3.23 or above.
```

Comment:
Yes, because 3.23 is what is specified as the minimum version in the `CMakeLists` files; I'm not aware of using any more recent features. But I think 3.26 was your suggestion in the last PR so perhaps you did find an issue?

Reply:
I had some issues with 3.23, but I am not sure that is something from my installation that had gone wrong or from some specific problems with the version. The problem with 3.23 was a crash without error message, so it was nothing that I could investigate in more details. Was it working with 3.23 for you?

Reply:
I see. I must admit I only tested with 3.26. I'm happy to change this to 3.26, and update the CMakeLists files as well.